src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -276,10 +276,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     Condition="'$(RazorDefaultConfiguration)'!=''">
 
     <ItemGroup>
-      <ResolvedRazorConfiguration Include="@(RazorConfiguration)" Condition="'%(RazorConfiguration.Identity)'=='$(RazorDefaultConfiguration)'" />
+      <ResolvedRazorConfiguration Include="@(RazorConfiguration->WithMetadataValue('Identity', '$(RazorDefaultConfiguration)')->Distinct())" />
```

Comment:
Doing a `Distinct` in case there's more than one `RazorConfiguration` present with the same value. You could get in to this if you reference a 2.0 package in a netcoreapp3.0 project. The scenario is clearly unsupported, but the error you get is not intuitive.

Reply:
👍